### ISSUE TITLE
Use @fnando/i18n-js to localization (i18n)

### ISSUE THREAD

**Issue by dirceupj** [ljagis/leaflet-measure#97]
I'm creating this Issue to discuss if the usage of https://github.com/fnando/i18n-js will better suit users needs there. 
Common application usage (at least mine) is to serve compiled assets and have more than one option of locale available in the app.

**Comment by brandoncopeland** [comment]
Yes. I'm open to other i18n solutions and open to hearing ideas from someone with more internationalization/localization experience than myself (almost none).

I would like to continue to keep small download size a priority as we move forward. That may influence our decision some.
